sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Listeners/ScanBlobScanLogHybridPollingStrategy.cs

```diff
@@ -282,7 +282,7 @@ private void ThrowIfDisposed()
             containerScanInfo.ContinuationToken = page.ContinuationToken;
 
             // if ending a cycle then copy currentSweepCycleStartTime to lastSweepCycleStartTime, if changed
-            if (page.ContinuationToken == null &&
+            if (string.IsNullOrEmpty(page.ContinuationToken) &&
```

Comment:
Above we're using string.IsNullOrWhiteSpace - we should be consistent. Also, do we have proof from logs that this is the actual issue?

Reply:
I was able to validate the fix locally.